ApollosProject/apollos-prototype
# Allows Username Login

## DESCRIPTION

<img width="463" alt="Screen Shot 2019-08-14 at 11 01 17 AM" src="https://user-images.githubusercontent.com/2659478/63037984-2eb8b080-be8e-11e9-9f84-8fe3320f9192.png">

### What does this PR do, or why is it needed?

This allows a user to login with a non-email string.

### What design trade-offs/decisions were made?

Navigating to `Auth` via `navigate` instead of `push`, seems to be fine. 🤷‍♂️

### How do I test this PR?

Change the value to false in the `NavigationService` and make sure you can login via username.

### What automated tests did you write?

None.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
